plugin.video.newyankeeworkshop/addon.xml

```diff
@@ -12,13 +12,17 @@
   <extension point="xbmc.addon.metadata">
     <summary lang="en_GB">Episodes of The New Yankee Workshop</summary>
     <description lang="en_GB">Hosted by master carpenter Norm Abram, who is legendary for his woodworking skills, The New Yankee Workshop has guided millions of viewers through the hands-on process of furniture making.</description>
-    <news>v0.0.1 (09.09.2018)
-	    - Initial Release</news>
+<news>
+v0.0.2 (05.17.2020)
+  - Updates to video extraction code
+v0.0.1 (09.09.2018)
+  - Initial Release
+</news>
     <language>en</language>
     <platform>all</platform>
     <license>GNU GENERAL PUBLIC LICENSE. Version 3, June 2007</license>
```

Comment:
Can you use a spdx identifier for the license? Should be GPL-3.0-only or GPL-3.0-or-later depending on your license

Reply:
Updated license to GPL-3.0-or-later ... thank you!!